src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -168,6 +168,12 @@ def create_feedback_issue(event, project_id, source: FeedbackCreationSource):
     if not project.flags.has_feedbacks:
         first_feedback_received.send_robust(project=project, sender=Project)
 
+    if (
+        source is FeedbackCreationSource.NEW_FEEDBACK_ENVELOPE
```

Comment:
would prefer `==` to `is` here as we generally restrict `is` comparisons to Singleton values like `None` etc. both valid, but i think we more consistently don't use `is` operator. https://stackoverflow.com/a/25858661

Reply:
```suggestion
        source == FeedbackCreationSource.NEW_FEEDBACK_ENVELOPE
```